WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/SelectableItemRow.swift

```diff
@@ -5,20 +5,21 @@ struct SelectableItemRow: View {
     let title: String
     let subtitle: String
     let selected: Bool
+    @Environment(\.isEnabled) var isEnabled
```

Comment:
Interesting, I didn't know there was an environment property to understand if the view is enabled or not.